Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorTransformComponentSelection.cpp

```diff
@@ -1557,7 +1546,8 @@ namespace AzToolsFramework
 
         struct SharedScaleState
         {
-            AZ::Vector3 m_savedScaleOffset = AZ::Vector3::CreateZero();
+            AZ::Vector3 m_savedScale = AZ::Vector3::CreateZero();
```

Comment:
I wonder if it would make more sense to initialize these to CreateOne() and 1.0f?

Reply:
Also, do you actually need a vector scale here any more now that Transform only supports uniform scale?

Reply:
You could be right 🤔 I'll have a look and see if it's worth updating 👍

Reply:
I am going to rename this back to `Offset` as it should not be initialized to `1.0f` but instead `0.0` as it's the _change_/_delta_, not absolute value